### PR TITLE
Change back the unit size from KB to kB for backward compatibility

### PR DESCRIPTION
**Pull request by kewang1024**

### CODE REVIEW

**Review by xiaoxmeng — APPROVED** [review]
@kewang1024 thanks!

File: velox/core/QueryConfig.cpp [hunk 1]
```diff
@@ -43,7 +43,8 @@ CapacityUnit valueOfCapacityUnit(const std::string& unitStr) {
   if (unitStr == "B") {
     return CapacityUnit::BYTE;
   }
-  if (unitStr == "KB") {
+  // To be backward compatible this is lowercase.
+  if (unitStr == "kB") {
```
Comment (spershin) [inline velox/core/QueryConfig.cpp:47]: Why don't we support all flavors of upper/lower cases?
Like tolower() it and then compare with lower case all time time?
@kewang1024 
And that's for all units.

Comment (kewang1024) [reply velox/core/QueryConfig.cpp:47]: That's a good idea. Let me put that out in the followup PR? Looks like other folks are blocked by this issue for now.

**Comment by tanjialiang** [comment]
Some tests are still failing

**Pull request closed without merging**